TechnicalDocuments/FastlaneMatch.md

```diff
@@ -0,0 +1,147 @@
+Fastlane Match
+==============
+
+Fastlane match is a tool for managing certificates and provisioning profiles centrally. It downloads certificates and profiles from a GitHub repository or Google Cloud Storage bucket. For safety, centrally stored files are encrypted.
+
+## Installing Certificates and Provisioning Profiles Locally
+
+For day to day development work it usually suffices to navigate to the project directory in a terminal shell and run the command
+
+```shell
+ $ fastlane match development --readonly
+```
+
+Certificates and provisioning profiles for enterprise builds (Hockey App) or release builds (Testflight and the App Store) can also be downloaded. Normally, this is not needed but might be necessary if you want to run UI tests locally or work on updating our fastlane build scripts.
+
+To install certificates and provisioning profiles for creating a Testflight or App Store build locally, run the command
+
+```shell
+ $ fastlane match release git_branch:"release team id" --read-only
+```
+
+For a Hockeyapp build
+
+```shell
+ $ fastlane match enterprise --readonly
+```
+
+More information about available options can be found below and in the fastlane match [documentation](https://docs.fastlane.tools/actions/match/).
+
+## Several Targets and Teams
+
+Recommended practise is to have on branch per team. It is also possible to create branches for different bundle identifiers should it get messy with a lot of app identifiers for one team. Default values for the git URL, branch, app identifier and build type are read from the `Matchfile`.
+
+It can happen that code signing certificates for more than one target or type are needed for a build. This can be handled by making several calls to match.
+
+```ruby
+match(app_identifier: "com.best.app.ever.seriously", type: "adhoc", git_branch: "best_app_ever_ad_hoc_signing")
+match(app_identifier: "com.best.app.ever.seriously", type: "development", git_branch: "best_app_ever_dev_signing")
+```
+
+Repository URLs can be found in the [new hires checklist](https://github.com/Babylonpartners/ios-playbook#4-new-hires-checklist). Information about which branches we use for different Babylon apps can be found in our internal Confluence Wiki.
+
+## Update Contents Manually
+
+Apple does not allow more than two distribution certificates per account, so it can happen that fastlane match cannot create a new distribution certificate and is unable to figure out which existing certificate to link with a new provisioning profile. Fastlane match provides a utility for nuking all existing certificates and provisioning profiles and replace them with new ones generated by fastlane match. This is, however, not always advisable. Nuking existing enterprise certificates can be a really bad idea as in house apps will stop working.
```

Comment:
It shouldn't really happen that we need more than one distribution certificate. I'd rephrase it and maybe would hide the whole section under `summary` tag as this is something we wouldn't ever need to do any more since we now fully manage our certificates via match.